proto/rill/admin/v1/api.proto

```diff
@@ -1127,6 +1127,14 @@ message GetGithubUserStatusResponse {
   string access_token = 3;
   string account = 4;
   repeated string organizations = 5;
+  bool is_user_app_installed = 6;
+  string user_installation_permission =7;
+  repeated OrganizationWithApp organizations_with_app = 8;
+}
+
+message OrganizationWithApp {
+  string org = 1;
+  string permission = 2;
 }
```

Comment:
What do you think about using `map<string, string> organization_installation_permissions` (doesn't require a separate type)?